docs/scu/dev/features/integrating-code-releases.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Integrating code releases
+description: Spryker Code Upgrader automatically integrates code releases
+template: concept-topic-template
+---
+
+Spryker architecture supports the extensions of modules using extension points in the project code. With Spryker Code Upgrader, the process of integrating plugins, configuration updates, adding new translations, and other code changes is automated. As a result, to add new features, developers don't need to apply code changes manually.
+
+The Upgrader supports the following extension points:
+- Configuration of the following:
+  - Module
+  - Environment
+- Copying module files
+- Adding and removing the following:
+  - Plugin
+  - Navigation element
+  - Widget
+  - Glue relationship
+- Adding the following:
+  - Transfer definition
+  - DB schema definition
+
+## Release integrability rating
+
+Release integrability rating is a measure of the integrability coverage of a code release. It represents the percentage of automatically integratable plugins, configurations, schemas, and other components.
+
+This rating is determined during the internal testing where the generated integration code is compared with the manually released code to assess the level of integration achieved. If both are equal, then the release is fully auto-integratable.
+
+There are types of releases that provide only dependency updates without any code integrations, so-called module-only releases. Such releases have 100% coverage.
+
+Each of the Upgrader's PRs contains the release rating of the suggested release.
```

Comment:
What should I do with the PRs that don't have 100% coverage?

Reply:
Nothing, we have the threshold to ignore release integration with the rating less than defined. Everything is installed automatically